### PR TITLE
Credit card fraud

### PR DESCRIPTION
**Pull request by msluszniak**
And some minor changes

### CODE REVIEW

File: notebooks/credit_card_fraud.livemd [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+  {:nx, "~> 0.2.1"},
+  {:explorer, "~> 0.2.0"}
+])
+
+EXLA.set_as_nx_default([:tpu, :cuda, :rocm, :host])
+```
+
+## Data processing
+
+The first step is to prepare the data for training and evaluation processes. We load the data set into an ```Explorer.DataFrame``` object from CSV. Then we split the data into training and test data (in proportion 80% into a training set and 20% into a test set) using the *split_train_test* function. In the next step, we separate features from target labels using *split_features_targets*. Now we should normalize the data to our neural-net performs better. The normalize_data function divides each value for a particular feature by the maximum absolute value in this feature. In the end, we cast data frames into tensors with *df_to_tensor*.
```
Comment (seanmor5) [inline notebooks/credit_card_fraud.livemd:24]:
```suggestion
The first step is to prepare the data for training and evaluation. We load the data set into an ```Explorer.DataFrame``` object from CSV. Then we split the data into training and test data (in proportion 80% into a training set and 20% into a test set) using the *split_train_test* function. In the next step, we separate features from target labels using *split_features_targets*. Now we should normalize the data to our neural-net performs better. The normalize_data function divides each value for a particular feature by the maximum absolute value in this feature. In the end, we cast data frames into tensors with *df_to_tensor*.
```
